style.css

```diff
@@ -0,0 +1,64 @@
+body {
+  font-family: 'Arial', sans-serif;
+  background-color: #f4f4f4;
+  margin: 0;
+  padding: 0;
+  direction: rtl;
+  line-height: 1.6;
+}
+
+main {
+  width: 80%;
+  max-width: 900px;
+  margin: 40px auto; /* Added some margin on top for better spacing */
+  background-color: white;
+  padding: 20px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1); /* Adjusted shadow for a softer look */
+  border-radius: 8px;
+}
+
+h1 {
+  text-align: center;
+  color: #333;
+  margin-bottom: 20px; /* Added margin bottom for spacing */
+}
+
+#img-div {
+  text-align: center;
+  margin: 20px 0;
+}
+
+#image {
+  width: 100%;
+  max-width: 400px;
+  height: auto;
+  border-radius: 8px;
+}
+
+#img-caption {
+  color: #666;
+  font-size: 14px;
+  margin-top: 10px;
+}
```

Comment:
- Your img element should have a display of block.
- Your #image should have a max-width of 100%.

Reply:
Hi sir!  thanks for your feedback, I will follow all my mistakes...

Reply:
Hi sir! I completed my 3rd project requirements but I don't know what is wrong with my survey form  project @ahzamir